cookbook/integration_llama-index_milvus-lite.ipynb

```diff
@@ -0,0 +1,235 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        uri="./milvus_demo.db", dim=1536, overwrite=False
</pre><p>Nit, "milvus_demo.db" is just fine, and simpler. In future I'll have better documentation to unify our examples with things like "./milvus_demo.db" v.s.  "milvus_demo.db".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/641/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
moved to tmp dir to be able to easily gitignore it in our cookbook dir, hope this is ok